content/docs/services/relational-database.md

```diff
@@ -13,16 +14,16 @@ If your application uses relational databases for storage, you can use the AWS R
 
 Plan Name | Description | Price
 --------- | ----------- | -----
-`shared-psql`            | Shared infrastructure for Postgres DB               | Free
-`medium-psql`            | Dedicated Medium RDS Postgres DB Instance           | $0.115 / hr + storage
-`medium-psql-redundant`  | Dedicated Redundant Medium RDS Postgres DB Instance | $0.230 / hr + storage
-`large-psql`             | Dedicated Large RDS Postgres DB Instance            | $0.230 / hr + storage
-`large-psql-redundant`   | Dedicated Redundant Large RDS Postgres DB Instance  | $0.470 / hr + storage
+`shared-psql`            | Shared infrastructure for PostgreSQL DB               | Free
```

Comment:
Let's spell out "database" instead of "DB" in all of these descriptions. But specifically for this one, "Shared infrastructure for PostgreSQL DB" is a bit vague - what is it really? Is it really "A shared PostgresSQL database for prototyping use (no production data)" ?